lib/count-total-users-by-year.rb

```diff
@@ -1,33 +1,26 @@
 require 'zendesk_api'
 
-@client = ZendeskAPI::Client.new do |config|
-  config.url = ENV['ZENDESK_URL']
-  config.username = ENV['ZENDESK_USER_EMAIL']
-  config.password = ENV['ZENDESK_USER_PASSWORD']
+require_relative 'zendesk-setup.rb'
 
-  config.retry = true
-end
-
-today = Date.today.next_day
-lastyear = today - 365
+lastyear = Date.today.next_day - 365
 
 puts "Total Users: #{@client.users.count}"
 
 puts "Total Users 2012"
-puts @client.search(:query => "type:user created_at>2012-01-01 created_at<2013-01-01").count
+puts @client.search(:query => "type:user created_at>2012-01-01 created_at<2013-01-01").count.to_i
```

Comment:
Can we add `=` to ensure that the query include 1st January?

```suggestion
puts @client.search(:query => "type:user created_at>=2012-01-01 created_at<2013-01-01").count.to_i
```

Reply:
There is no need here @adityapahuja because we didn't actually start using Zendesk until well into the year, maybe April. I picked 1 Jan as an arbitrary date :-)

Reply:
Resolved as suggested.